atrium-core/src/commonMain/kotlin/ch/tutteli/atrium/creating/impl/BaseExpectImpl.kt

```diff
@@ -63,8 +63,7 @@ abstract class BaseExpectImpl<T>(
             representationInsteadOfFeature?.let { provider ->
                 maybeSubject.fold({ null }) { provider(it) }
             } ?: maybeSubject.getOrElse {
-                // a RootExpect without a defined subject is almost certain a bug
-                Text(SHOULD_NOT_BE_SHOWN_TO_THE_USER_BUG)
+                Text.EMPTY
```

Comment:
Please don't change this in the end, if you saw this text in reporting, then it is most likely an implementation bug :wink:

Reply:
Well, what is your recommendation then?
Currently, Atrium uses an explicit `None` subject in order to render `elements need all:` block.: https://github.com/robstoll/atrium/blob/65227add3c31604a64cf4ef86671eadc78be3bde/logic/atrium-logic/src/commonMain/kotlin/ch/tutteli/atrium/logic/impl/containsHelpers.kt#L45